models/classes/LtiRoles.php

```diff
@@ -26,9 +26,15 @@
  */
 interface LtiRoles
 {
-    const CONTEXT_LEARNER = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Learner';
+    const PROPERTY_CONTEXT_LEARNER = 'http://www.imsglobal.org/imspurl/lis/v1/vocab/membership#Learner';
```

Comment:
this is not really a property + it is used in various context so we won't change it

Reply:
I have adjust the name and moved to other extension since it's used there (extension-tao-ltideliveryprovider)